### PR TITLE
feat: created tooltip

### PR DESCRIPTION
**Pull request by nurovigor**
Added a tooltip to display the information of the selected table

### CODE REVIEW

File: src/components/atoms/Tooltip/styles.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+`;
+
+export const TooltipTip = css`
+	position: absolute;
+	border-radius: 4px;
+	left: 50%;
+	transform: translateX(-50%);
+	padding: 6px;
+	color: black;
+	background: #f1f1f1;
```
Comment (a-polikarpov) [inline src/components/atoms/Tooltip/styles.ts]: Use a constant in the future

File: src/hooks/useBlockPosition.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from 'react';
+
+export const useBlockPosition = (ref: any) => {
```
Comment (a-polikarpov) [inline src/hooks/useBlockPosition.ts]: What is it?

**Review by a-polikarpov — CHANGES_REQUESTED** [review]

File: src/hooks/useBlockPosition.ts (hunk 2 above)
Comment (nurovigor) [reply src/hooks/useBlockPosition.ts]: I created a hook to encapsulate the logic for calculating the window size and tooltip, I will remove it

File: src/components/atoms/Tooltip/styles.ts (hunk 1 above)
Comment (nurovigor) [reply src/components/atoms/Tooltip/styles.ts]: Oke

**Review by a-polikarpov — APPROVED** [review]

**Pull request merged**